### ISSUE TITLE
Import parallel fails with "Cannot find main entry point"

### ISSUE THREAD

**Issue by benbucksch** [parallel-js/parallel.js#187]
Reproduction:
1. Install: `yarn add https://github.com/parallel-js/parallel.js`
2. Code: `import Parallel from 'paralleljs';`
3. Run: `node --experimental-modules myapp.js` (ESM experimental, using node v13.2.0)

Actual result:
Error: "Cannot find main entry point" in package.json

```
(node:1010476) ExperimentalWarning: The ESM module loader is experimental.
internal/modules/esm/default_resolve.js:94
  let url = moduleWrapResolve(specifier, parentURL);
            ^

Error: Cannot find main entry point for .../node_modules/paralleljs/ imported from .../myapp.js
    at Loader.resolve [as _resolve] (internal/modules/esm/default_resolve.js:94:13)
    at Loader.resolve (internal/modules/esm/loader.js:74:33)
    at Loader.getModuleJob (internal/modules/esm/loader.js:148:40)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:41:40)
    at link (internal/modules/esm/module_job.js:40:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
error Command failed with exit code 1.
```

Expected result:
Imports and runs

**Comment by benbucksch** [comment]
The fix is trivial: Add
```
  "main": "lib/parallel.js",
```
to package.json

**Comment by benbucksch** [comment]
Fixed PR #188

**Comment by benbucksch** [comment]
PR was merged. Thanks, @mathiasrw !
